src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderService.java

```diff
@@ -2653,6 +2653,7 @@ public void setHearingSelectedInfoForTask(List<Element<HearingData>> ordersHeari
             List<HearingData> hearingList = ordersHearingDetails.stream()
                 .map(Element::getValue).toList();
 
+
```

Comment:
@JamesCutts00  can you please remove this change. as it does not have any significance of change.

Reply:
@yogendraU  I have removed the space